fns-hl7-pipeline/fn-redactor/src/main/kotlin/gov/cdc/dex/hl7/Function.kt

```diff
@@ -69,15 +66,18 @@ class Function {
                     routeElement.asString
                 }
 
-                logger.info("DEX:: Received and Processing messageUUID: $messageUUID, filePath: $filePath")
+                context.logger.info("DEX:: Received and Processing messageUUID: $messageUUID, filePath: $filePath")
```

Comment:
according to latest changes on logging, we should be using our own logger from LogFactory.getLogger(0 instead of context.logger.

Reply:
Just pushed the fix.  Commit id: 31c1a6c